templates/website/about-privacy.html

```diff
@@ -11,183 +11,288 @@
+                            send you a reminder after 24 hours;</li>
+                            <li><strong>b)</strong> Two weeks after you send your message, we send you an automated email
+                            to ask whether you received a response. If you do not respond, we send
+                            you a single reminder.</li>
+                            <li><strong>c)</strong> On rare occasions, we pass on a message from a politician who is
+                            concerned that they have mislaid emails from their constituents,
+                            inviting you to re-establish contact.</li>
+                        </ul>
+                    </li>
+                </ul>
+
+                <h3>
+                    Email opt-out
+                </h3>
+
+                <p>
+                    As part of the process of sending your message to your representative we
+                    will send you a limited number of emails, as described above. If during this
+                    process you wish to stop receiving emails from us, or you are a
+                    representative who no longer wishes to receive email via WriteToThem*,
+                    please contact us at <a href="mailto:<?= OPTION_CONTACT_EMAIL ?>"><?= $contact_email ?></a>.
+                </p>
+
+                <p>
+                    * If you are a representative wishing to opt out of WriteToThem, we
+                    encourage you to think again! WriteToThem is a system which makes it simple
+                    for your constituents to contact you, where they might find other channels
+                    more difficult to access. We have measures in place that prevent bulk
+                    mailings and contact with representatives other than the constituents&rsquo; own.
+                    You may wish to read more about the service and how you can best use it at
+                    <a href="https://www.mysociety.org/mp-services/">https://www.mysociety.org/mp-services/</a>.
+                </p>
+
+                <h3>
+                    Legal basis for processing
+                </h3>
+
+                <p>
+                    In using WritetoThem to contact a representative, you are consenting to the
+                    processes as described on this page. The legal basis is  6(1)(a) – Consent
+                    of the data subject.
+                </p>
+
+                <h3>
+                    Retention policies
+                </h3>
+
+                <ul>
+                    <li>
+                        We store your IP address and the pages that you visited whilst on the
+                        site as part of our site logs. We keep these for one month.
+                    </li>
+                    <li>
+                        We do not keep permanent copies of the letters you write to send to your
+                        representatives.
+                    </li>
```

Comment:
Not sure why above/below are separate bullet points.

Reply:
Clarity I guess?